net/pfSense-pkg-freeradius2/files/usr/local/pkg/freeradius.inc

```diff
@@ -4517,6 +4531,15 @@ function freeradius_validate_eap($post, &$input_errors) {
 		$input_errors[] = "The 'Maximum Sessions Tracking Per Server' field must contain an integer value.";
 	}
 
+	// Certificates for TLS
+	if ($post['ssl_ca_cert'] == 'none') {
```

Comment:
Well probably because showing random first thing returned by the lookup did not seem like the best idea around? (Generic problem with these drop-downs in pfS.) Been there for ages, personally I have no clue what's been the idea with this. The package has never worked without certificates, originally it's been bootstrapping some self-signed one there using the removed bundled certificate "manager". 

It could be renamed to "not configured" or even " " but there needs to be some default empty placeholder, otherwise users will end up saving a random cert/CA combo that doesn't match each other.